egs/swbd/s5c/local/ctc/run_lstm.sh

```diff
@@ -22,7 +22,7 @@ norm_based_clipping=true
 # natural gradient options
 ng_per_element_scale_options=
 ng_affine_options=
-num_epochs=4
+num_epochs=2
```

Comment:
2 epochs is good for tuning, but we may  need to increase it later on for best results.. no problem to merge though.